todos/week3/4_classestodo.py

```diff
@@ -15,14 +17,16 @@
 # TODO: of Employee set equal to a variable employeeOne
 
 
-
+# TODO: ###########################################################################################
+# TODO: Section 3:
```

Comment:
There should be a purpose to this section as well, like a suggestion of "Now let's add a Contractor child class to the Employee class. Contractors are like employees, but they don't have all the benefits of one. Since our Employee class has benefits [add in benefits to the Employee class before inserting this change, Mark], we will want to instantiate our Contractor classes with no benefits. However, we will want a new attribute for our child class. We want "contract_length" so our Contractor class instantiations can specify how long a Contractor will be with the company.